src/commands/generate/serviceGenerator.ts

```diff
@@ -104,7 +108,8 @@ export function generateService(
             imports.push(...IType.visit(endpointDefinition.returns, importsVisitor));
         }
 
-        const signature: MethodDeclarationStructure = {
+        const signature: MethodSignatureStructure = {
```

Comment:
Why is this changing from a `MethodDeclaration` to a `MethodSignature`? Playing around with ast explorer it seems like a `MethodDeclaration` is inside a class, but a `MethodSignature` is inside an interface (but maybe our output here would be the same?)

<img width="1068" alt="image" src="https://user-images.githubusercontent.com/21962852/223368273-ca74a9d9-2415-4df6-9c2b-418461e86e91.png">

<img width="1095" alt="image" src="https://user-images.githubusercontent.com/21962852/223368391-9b50245f-e864-4748-bae8-815adce51af9.png">

Reply:
Looks like the output changes are purely comment whitespace, but still curious if there's a reason for the change here 🙂

Reply:
Ah I see, this is actually for an interface: https://github.com/palantir/conjure-typescript/pull/247/files/d4039eb8a88f298863ff81a1d76c254bf4a67d32..ecdeaa4669a7503f98d0b3babb85bd3cdb5e13a2#diff-cab300c03b8599d9d8e1740404123aea0e80562a8341a8c0813c77e22ebc17c5R144-R147